src/main/scala/uk/gov/hmrc/play/http/HeaderNames.scala

```diff
@@ -59,12 +61,15 @@ object HeaderNames {
     googleAnalyticTokenId,
     googleAnalyticUserId,
     deviceID, // not a typo, should be ID
-    akamaiReputation
+    akamaiReputation,
+    hmrcLang
   )
 }
 
 object CookieNames {
   val deviceID = "mdtpdi"
+  val hmrcLang = "HMRC_LANG"
```

Comment:
Similar to the header naming comment, please could we use something like `mdtp_lang` instead of `HMRC_LANG`. It makes it easier to find cookies if they all start with `mdtp`.